docs/troubleshooting.md

```diff
@@ -121,6 +122,15 @@ Some of the common reasons we have encountered that caused this `MESSAGE` are qu
 
 If you are trying to [install an operator from the cli](https://github.com/operator-framework/operator-marketplace#installing-an-operator-using-marketplace), the packages listed in the CatalogSourceConfig must have valid package names. If you have private app-resgistries, please follow the steps listed [here](https://github.com/operator-framework/operator-marketplace/blob/master/docs/how-to-authenticate-private-repositories.md).
 
+## Conflicting Package Names
+
+The Marketplace operator does not support reusing package names between two OperatorSources. To avoid unexpected behavior users should avoid adding any OperatorSources that reuse existing package names.
```

Comment:
Please use `Marketplace Operator` (instead of `operator`) everywhere to be consistent with the rest of the document.

Reply:
Will do.